homeassistant/components/knx.py

```diff
@@ -248,3 +287,60 @@ def __init__(self, hass, device, hook, action, counter=1):
             hass.data[DATA_KNX].xknx, self.script.async_run,
             hook=hook, counter=counter)
         device.actions.append(self.action)
+
+
+class KNXExposeTime(object):
+    """Object to Expose Time/Date object to KNX bus."""
+
+    def __init__(self, xknx, expose_type, address):
+        """Initialize of Expose class."""
+        self.xknx = xknx
+        self.type = expose_type
+        self.address = address
+        self.device = None
+
+    @callback
+    def async_register(self):
+        """Register listener."""
+        from xknx.devices import DateTime, DateTimeBroadcastType
+        broadcast_type_string = self.type.upper()
```

Comment:
Can this be done in the validation schema? Or do you need type to be different from broadcast_type?

Reply:
`self.type` is used within `KNXExposeTime` to look up the right entry within `DateTimeBroadcastType`  -Enum and within `ExposeSensor` which needs a lower case value.